developer-docs-site/docs/guides/aptos-apis.md

```diff
@@ -40,7 +40,9 @@ View functions do not modify blockchain state. A [View](https://github.com/aptos
 
 The View function operates similar to the [Aptos Simulation API](./system-integrators-guide.md#testing-transactions-or-transaction-pre-execution), though with no side effects and a accessible output path. The function is immutable if tagged as `#[view]`, the compiler will confirm it so and if fail otherwise. View functions can be called via the `/view` endpoint. Calls to view functions require the module and function names along with input type parameters and values.
 
-In order to use the View functions, you need to pass `--bytecode-version 6` to the Aptos CLI.
+In order to use the View functions, you need to pass `--bytecode-version 6` to the [Aptos CLI](../cli-tools/aptos-cli-tool/index.md) when publishing the module.
+
+> Note: Calling View functions is not yet supported by the Aptos CLI.
```

Comment:
This new note seems to contradict the statement make immediately above. Which CLI? I added Aptos yet am not certain.

Reply:
yeah, the aptos cli.